doc/tools/west/index.rst

```diff
@@ -57,17 +62,38 @@ West was introduced to unify and streamline the developer's experience when
 working with Zephyr. It is very important to note that west does not
 replace the CMake-based build system, but directly uses it and augments it.
 
+See :ref:`west-flash-debug` for a detailed explanation of the flash and debug
+command-line interface exposed by west.
+
 .. _west-struct:
 
 Structure
 *********
 
+West structure
+==============
+
 West is downloaded and installed on a system in two stages:
 
 * Bootstrapper: Installed using ``pip``, implements ``west init``
 * Installation: Installed using ``west init``, implements all other commands
 
-Additional information about the two parts can be found below.
+Additional information about the two distinct parts of west can be found in
+the corresponding sections below.
+
+Repository structure
+====================
+
+Beyond west itself, the actual code repositories that west works with
+are the following:
+
+* Manifest repository: Cloned by ``west init``, and managed afterward with Git
+  by the user. Contains the list of projects in the manifest :file:`west.yml`
+  file. In the case of upstream Zephyr, the zephyr repository is the manifest
+  repository.
+* Projects: Cloned and managed by ``west update``. Listed in the manifest file.
```

Comment:
You've got a link at the bottom of the page, so I suspect you intended this to read:
```
Listed in the `manifest file`_.
```

Reply:
Except that this sis generic, and the manifest file I link to is the specific one for Zephyr. I think it would be confusing for the user if I did that

Reply:
OK, i noticed the unreferenced link at the bottom of the files.  LGTM then.